src/stories/Library/banner/banner.scss

```diff
@@ -0,0 +1,65 @@
+@mixin banner-height {
+  height: 500px;
+  @include media-query__medium {
+    height: 810px;
+  }
+}
+
+@mixin banner-text-margin {
+  margin-bottom: $s-md;
+  @include media-query__medium {
+    margin-bottom: $s-xl;
+  }
+}
+
+.banner {
+  @include layout-container($layout__max-width--large, 0);
+  @include banner-height;
+  display: grid;
+  grid-template-rows: 1fr auto 1fr;
+  text-decoration: none;
+}
+
+.banner-content {
+  padding: $s-xl;
+  max-width: 375px;
+  background-color: white;
+  grid-row: 2;
+  grid-column: 1;
+
+  @include media-query__medium {
+    padding: $s-2xl;
+    max-width: 575px;
+    margin-left: 250px;
```

Comment:
I personally don't think this is a good approach to positioning the text container. The value of 250px might be correct in the figma example, but as the screen changes i would argue this should change too, and in that case this isn't responsive

I would suggest you to place the text inside a container with a smaller layout-container, and align that left side to make it move responsively.